4/4ti2/build_tarballs.jl

```diff
@@ -0,0 +1,96 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "4ti2"
+version = v"1.6.9"
+
+# Collection of sources required to build 4ti2
+sources = [
+    ArchiveSource("https://github.com/4ti2/4ti2/releases/download/Release_$(version.major)_$(version.minor)_$(version.patch)/4ti2-$(version).tar.gz",
+                  "3053e7467b5585ad852f6a56e78e28352653943e7249ad5e5174d4744d174966"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/4ti2-*
+
+ # Remove misleading libtool files 
+ rm -f ${prefix}/lib/*.la 
+ rm -f /opt/${target}/${target}/lib*/*.la 
+ rm -f /opt/${MACHTYPE}/${MACHTYPE}/lib*/*.la 
+
+# Patch #1 for fixing cross-compilation: The correctness of the
+# patch relies on us using clang or GCC (in a new enough version) as compiler;
+# otherwise we ought to turn off ftrapv support to be on the safe side.
+atomic_patch -p1 ../patches/ftrapv.patch
+
+# Patch #2 for fixing cross-compilation: The code detecting presence and usability
+# of GMP is broken for cross compilation; fix that
+atomic_patch -p1 ../patches/gmp.patch
+
+# Patch to fix compilation on mingw32: add missing #include <time.h>
+atomic_patch -p1 ../patches/time.patch
+
+
+./configure --prefix=$prefix --host=$target --enable-shared --disable-static --with-gmp=${prefix} --with-glpk=${prefix}
```

Comment:
Adding the build architecture should fix the errors on musl:
```suggestion
./configure --prefix=$prefix --host=$target --build=${MACHTYPE} --enable-shared --disable-static --with-gmp=${prefix} --with-glpk=${prefix}
```

Reply:
Thanks Benjamin!